{
  "comments": [
    {
      "key": {
        "uuid": "19cafe67_148a5b18",
        "filename": "main/src/main/java/org/onap/policy/pap/main/rest/PdpGroupQueryControllerV1.java",
        "patchSetId": 1
      },
      "lineNbr": 100,
      "author": {
        "id": 3239
      },
      "writtenOn": "2021-09-02T14:43:24Z",
      "side": 1,
      "message": "Wondering if we need to additionally add RuntimeException, or database provider layer handles it for all cases and throws PfModelRuntimeException instead.",
      "range": {
        "startLine": 100,
        "startChar": 42,
        "endLine": 100,
        "endChar": 65
      },
      "revId": "347bc1022aba20ae134848ef1a04930e9cb69b09",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": true
    }
  ]
}